packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -166,7 +165,7 @@ export interface SubnetSelection {
    *
    * At most one of `subnetType` and `subnetGroupName` can be supplied.
    *
-   * @default SubnetType.PRIVATE
+   * @default SubnetType.PRIVATE (or ISOLATED or PUBLIC if there are no PRIVATE subnets)
```

Comment:
I'm not sure this description of `@default` is super clear... `ISOLATED or PUBLIC`? I don't think this is a valid value... Do you mean whichever value resolves to a subnet first in that order?

Reply:
Yes, @rix0rrr I like you proposed change (and the value should be `-`). It should be something like:

```ts
/**
 * @default - the default subnet type will be the first available type in the following order: PRIVATE, ISOLATE, PUBLIC
 */

Something like that.